tests/attr/test_shapley.py

```diff
@@ -27,6 +27,18 @@ def test_simple_shapley_sampling(self) -> None:
             n_samples=250,
         )
 
+    # Test above, but with inputs of type long.
+    def test_sss_with_long_tensor_inputs(self) -> None:
+        net = BasicModel_MultiLayer()
+        inp = torch.tensor([[20, 50, 30]], dtype=torch.int64)
```

Comment:
I think the float conversion added above is needed for this test to pass since the linear layer in BasicModel_MultiLayer expects a float tensor input. It would be better to update these tests with a model that accepts a Long input, which would be representative of cases where the user provides a Long input for the model.  One option might be BasicModel, it returns an output with the same type as the provided input and is used for the int tests in #207 as well. Just 1-2 cases should be sufficient, ideally covering both long input with long output and long input with float output.